### PR TITLE
fix(tests): fail on non-existing environment variable; 'LocalOutput' mention

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: tests/test_vcs.py [hunk 1]
```diff
@@ -37,7 +37,10 @@ def test_git_success_command_line_poll_no_changes(stdout_checker, git_server, tm
 
 
 def test_vcs_error_no_vcs_type(capsys):
-    del os.environ["VCS_TYPE"]
+    try:
+        del os.environ["VCS_TYPE"]
```
Comment (i-keliukh) [inline tests/test_vcs.py]: Why this change is needed?

Comment (k-dovgan) [reply tests/test_vcs.py]: Because if system _does not_ have a default VCS type claimed in environment (unlike our current CI system), this line fails with KeyError. So the variable should be unset _if present._
